packages/web-components/fast-foundation/src/anchored-region/anchored-region.ts

```diff
@@ -18,13 +18,53 @@ declare global {
     }
 }
 
-export type AxisPositioningMode = "uncontrolled" | "locktodefault" | "dynamic";
-export type AxisScalingMode = "anchor" | "fill" | "content";
-export type GbcrUsage = "always" | "never" | "default";
-export type HorizontalPosition = "start" | "end" | "left" | "right" | "unset";
-export type VerticalPosition = "top" | "bottom" | "unset";
+/**
+ * Defines the base behavior of an anchored region on a particular axis
+ *
+ * @beta
+ */
+export enum AxisPositioningMode {
+    uncontrolled = "uncontrolled",
+    locktodefault = "locktodefault",
+    dynamic = "dynamic",
+}
 
-export interface Dimension {
+/**
+ * Defines the scaling behavior of an anchored region on a particular axis
+ *
+ * @beta
+ */
+export enum AxisScalingMode {
+    anchor = "anchor",
+    fill = "fill",
+    content = "content",
+}
+
+/**
+ * Defines the horizontal positioning options for an anchored region
+ *
+ * @beta
+ */
+export enum HorizontalPosition {
+    start = "start",
+    end = "end",
+    left = "left",
+    right = "right",
+    unset = "unset",
+}
+
+/**
+ * Defines the vertical positioning options for an anchored region
+ *
+ * @beta
+ */
+export enum VerticalPosition {
+    top = "top",
+    bottom = "bottom",
+    unset = "unset",
+}
+
+interface Dimension {
```

Comment:
Should this be exported?

Reply:
Also, should we include typedoc info here? If it's exported - is it public or internal?

Reply:
marked as internal